<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ConfigModel</class>
 <widget class="QDialog" name="Dialog_ConfigModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config Item</string>
  </property>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Information</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>231</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Item Name (*)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ItemName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Money (*)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Money">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Employee (*)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Employee"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>(*) : Input is requirement</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Insert">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_itemList">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>511</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Update">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clear">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_ItemName</tabstop>
  <tabstop>lineEdit_Money</tabstop>
  <tabstop>comboBox_Employee</tabstop>
  <tabstop>pushButton_Clear</tabstop>
  <tabstop>pushButton_Insert</tabstop>
  <tabstop>pushButton_Update</tabstop>
  <tabstop>pushButton_Delete</tabstop>
  <tabstop>tableView_itemList</tabstop>
  <tabstop>pushButton_Close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_ConfigModel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
